lib/fathead/less/output.txt

```diff
@@ -0,0 +1,84 @@
+% format	A			string functions						http://lesscss.org/public/img/logo.png	<section class="prog__container"><p>The function <code>%(string, arguments ...)</code> formats a string.</p><p>The first argument is string with placeholders. All placeholders start with percentage symbol <code>%</code> followed by letter <code>s</code>,<code>S</code>,<code>d</code>,<code>D</code>,<code>a</code>, or <code>A</code>. Remaining arguments contain expressions to replace placeholders. If you need to print the percentage symbol, escape it by another percentage <code>%%</code>.</p><p>Use uppercase placeholders if you need to escape special characters into their utf-8 escape codes.The function escapes all special characters except <code>()&apos;~!</code>. Space is encoded as <code>%20</code>. Lowercase placeholders leave special characters as they are.</p><span class="prog__sub">Placeholders</span><p></p><ul class="prog__ul"><li><code>d</code>, <code>D</code>, <code>a</code>, <code>A</code> - can be replaced by any kind of argument (color, number, escaped value, expression, ...). If you use them in combination with string, the whole string will be used - including its quotes. However, the quotes are placed into the string as they are, they are not escaped by &quot;/&quot; nor anything similar.</li><li><code>s</code>, <code>S</code> - can be replaced by any expression. If you use it with string, only the string value is used - quotes are omitted.</li></ul><span class="prog__sub">Parameters</span><p></p><ul class="prog__ul"><li><code>string</code>: format string with placeholders,</li><li><code>anything</code>* : values to replace placeholders.</li></ul><span class="prog__sub">Returns</span><p> formatted <code>string</code>.</p><span class="prog__sub">Example</span><p></p><pre><code class="lang-less">format-a-d: %(&quot;repetitions: %a file: %d&quot;, 1 + 2, &quot;directory/file.less&quot;);\nformat-a-d-upper: %(&apos;repetitions: %A file: %D&apos;, 1 + 2, &quot;directory/file.less&quot;);\nformat-s: %(&quot;repetitions: %s file: %s&quot;, 1 + 2, &quot;directory/file.less&quot;);\nformat-s-upper: %(&apos;repetitions: %S file: %S&apos;, 1 + 2, &quot;directory/file.less&quot;);</code></pre><span class="prog__sub">Output</span><p></p><pre><code class="lang-css">format-a-d: &quot;repetitions: 3 file: &quot;directory/file.less&quot;&quot;;\nformat-a-d-upper: &quot;repetitions: 3 file: %22directory%2Ffile.less%22&quot;;\nformat-s: &quot;repetitions: 3 file: directory/file.less&quot;;\nformat-s-upper: &quot;repetitions: 3 file: directory%2Ffile.less&quot;;</code></pre></section>	http://lesscss.org/functions/#string-functions-% format
```

Comment:
`string functions` is a little too generic. The categories have no context so we need more specific names. Let's use `LESS String Functions` (Title Casing would be nice too)